diagnostic.md

```diff
@@ -94,4 +94,4 @@ var teamMember = {
 What is the value of `teamMember['hobby']`?  Of `teamMember['name']`?  Of
 `teamMember['name']['surname']`?
 
-_Replace this text with your answer._
+exercise, Jason Weeks, Weeks
```

Comment:
2 out of 3.

`teamMember['name']` should equal `{ given: 'Jason', surname: 'Weeks'};`
 This is because name is an object which has keys and values that are returned when `teamMember['name']` if called .